fuel-coroutines/build.gradle.kts

```diff
@@ -1,16 +1,13 @@
 import com.novoda.gradle.release.PublishExtension
 import org.jetbrains.kotlin.gradle.dsl.Coroutines
 import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
-import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
 
-plugins { java }
+plugins { java
+    kotlin("jvm")
```

Comment:
I think kotlin plugin is already applied at the `build.gradle` at the root level? I don't think we need it here?